src/top_model/main.cpp

```diff
@@ -211,5 +213,12 @@ std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> TOP = std::make_share
     r.run_until(NDTime("04:00:00:000"));
     auto elapsed = std::chrono::duration_cast<std::chrono::duration<double, std::ratio<1>>>(hclock::now() - start).count();
     cout << "Simulation took:" << elapsed << "sec" << endl;
+
+    FILE *input;

```

Comment:
It is a suggestion for better understanding, you can add comments for file pointers and function called as this part is newly added. Same can be implemented in all main.cpp files.

Reply:
Thanks,
It has been resolved.